packages/web-components/fast-components/src/picker/picker-list.styles.ts

```diff
@@ -0,0 +1,83 @@
+import { css, ElementStyles } from "@microsoft/fast-element";
+import {
+    ElementDefinitionContext,
+    forcedColorsStylesheetBehavior,
+    FoundationElementDefinition,
+} from "@microsoft/fast-foundation";
+import { SystemColors } from "@microsoft/fast-web-utilities";
+import {
+    accentFillActive,
+    accentFillRest,
+    bodyFont,
+    controlCornerRadius,
+    designUnit,
+    focusStrokeOuter,
+    neutralFillInputHover,
+    neutralFillInputRest,
+    neutralForegroundRest,
+    strokeWidth,
+    typeRampBaseFontSize,
+    typeRampBaseLineHeight,
+} from "../design-tokens";
+import { heightNumber } from "../styles/index";
+
+export const pickerListStyles: (
+    context: ElementDefinitionContext,
+    definition: FoundationElementDefinition
+) => ElementStyles = (
+    context: ElementDefinitionContext,
+    definition: FoundationElementDefinition
+) =>
+    css`
+        .picker-list {
+            display: flex;
+            flex-direction: row;
+            column-gap: calc(${designUnit} * 1px);
+            row-gap: calc(${designUnit} * 1px);
+            flex-wrap: wrap;
+            z-index: 1000;
+        }
+
+        [role="combobox"] {
+            min-width: 260px;
+            width: auto;
+            box-sizing: border-box;
+            color: ${neutralForegroundRest};
+            background: ${neutralFillInputRest};
+            border-radius: calc(${controlCornerRadius} * 1px);
+            border: calc(${strokeWidth} * 1px) solid ${accentFillRest};
+            height: calc(${heightNumber} * 1px);
+            font-family: ${bodyFont};
+            outline: none;
+            user-select: none;
+            font-size: ${typeRampBaseFontSize};
+            line-height: ${typeRampBaseLineHeight};
+            padding: 0 calc(${designUnit} * 2px + 1px);
```

Comment:
is the 1px addition to account for stroke width?

Reply:
This matches the styles for text field and number field which have similar inputs.  Maybe it guarantees there will always be at least one pixel space between the caret and the edge of the box?
https://github.com/microsoft/fast/blob/4b0ed1358c9a5d694908abdfebca6ef32607c29b/packages/web-components/fast-components/src/text-field/text-field.styles.ts#L66